plasma_core/block.py

```diff
@@ -1,20 +1,21 @@
-import rlp
-from rlp.sedes import binary, CountableList, big_endian_int
+from rlp.sedes import CountableList, big_endian_int
+from rlp.sedes.serializable import Serializable
 from ethereum import utils
 from plasma_core.utils.signatures import sign, get_signer
 from plasma_core.utils.merkle.fixed_merkle import FixedMerkle
 from plasma_core.transaction import Transaction
 from plasma_core.constants import NULL_SIGNATURE
 
 
-class Block(rlp.Serializable):
-
+class UnsignedBlockRLP(Serializable):
```

Comment:
I think we'll need an RLP with block and and signatures down the road, what do you think of creating the classes `Block` and `SignedBlock` and having SignedBlock inherit from `Block`?